src/test/java/org/ladocuploader/app/journeys/LaDigitalAssisterFlowJourneyTest.java

```diff
@@ -86,53 +88,43 @@ void hourlyIncomeFlow() {
     assertThat(testPage.getTitle()).isEqualTo(message("job-hourly-wage.title"));
     testPage.clickContinue();
 
-    assert (testPage.hasErrorText(message("error.missing-dollar-amount")));
+    assertTrue(testPage.hasErrorText(message("error.missing-dollar-amount")));
     testPage.enter("hourlyWage", "a");
     testPage.clickContinue();
 
-    assert (testPage.hasErrorText(message("error.invalid-money")));
+    assertTrue(testPage.hasErrorText(message("error.invalid-money")));
     testPage.enter("hourlyWage", ".99");
 
     testPage.clickContinue();
 
     assertThat(testPage.getTitle()).isEqualTo(message("job-hours-per-week.title"));
     testPage.clickContinue();
 
-    assert (testPage.hasErrorText(RANGE_ERROR_MESSAGE));
-    assert (testPage.hasErrorText(message("error.missing-general")));
+    assertTrue(testPage.hasErrorText(RANGE_ERROR_MESSAGE));
+    assertTrue(testPage.hasErrorText(message("error.missing-general")));
```

Comment:
Above we changed message calls to hard coded string values, but here we still use message calls. Is there a preferred pattern? Would it not be easier to use message calls in case the strings in those message properties change?

Reply:
That's fair, I don't have a strong opinion one way or the other, but I do think we should be consistent.